recipe-client-addon/lib/RecipeRunner.jsm

```diff
@@ -39,88 +39,147 @@ Cu.importGlobalProperties(["fetch"]);
 this.EXPORTED_SYMBOLS = ["RecipeRunner"];
 
 const log = LogManager.getLogger("recipe-runner");
-const prefs = Services.prefs.getBranch("extensions.shield-recipe-client.");
```

Comment:
I did this because we are accessing prefs from more than just the `extensions.shield-recipe.client` sub-tree. It seemed awkward to use `prefs.getCharPref` in someplaces and `Services.prefs.getCharPref` in others, depending on which preference was being accessed.